src/bioregistry/data/bioregistry.json

```diff
@@ -100737,6 +100737,33 @@
     ],
     "uri_format": "http://purl.obolibrary.org/obo/SBO_$1"
   },
+  "scancer": {
+    "contact": {
+      "email": "zty2009@hit.edu.cn",
+      "name": "Tianyi Zhao",
+      "orcid": "0000-0001-7352-2959"
+    },
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "SCancerRNA database contains experimentally supported data on long ncRNA, microRNA, PIWI-interacting RNA, small nucleolar RNA, and circular RNA biomarkers, as well as data on their differential expression at the cellular level.",
+    "example": "9530",
+    "homepage": "http://www.scancerrna.com/BioMarker/",
+    "name": "SCancerRNA",
+    "pattern": "^\\d{1,4}$",
+    "publications": [
+      {
+        "doi": "10.1093/gpbjnl/qzae023",
+        "pubmed": "39341795",
+        "title": "SCancerRNA: Expression at the Single-cell Level and Interaction Resource of Non-coding RNA Biomarkers for Cancers",
+        "year": 2024
+      }
+    ],
+    "uri_format": "http://www.scancerrna.com/BioMarker/toDetail?id=$1"
```

Comment:
the curation of the prefix should be about the entity type that the prefix describes. Here, this is corresponding to a biomarker in this database. The description of a bioregistry record should describe the semantic space, not the database.

Reply:
It turns out the `BioMarker` suffix is redundant, all of this resource is about biomarkers and the URLs work without the extra BioMarker suffix.

Reply:
updated the description to better describe the semantic space.